sonar-db/src/test/resources/org/sonar/db/duplication/DuplicationDaoTest/insert-result.xml

```diff
@@ -15,8 +15,7 @@
   <projects id="1" uuid="1" kee="foo" enabled="1" scope="FIL" qualifier="CLA"/>
 
   <duplications_index id="1"
-                      project_snapshot_id="1"
-                      snapshot_id="2"
+                      analysis_uuid="u1"
```

Comment:
I find alignment in DBUnit files is helpful since attributes will not be on the same column depending of the table name, it helps readability of the file.
let's discuss it, indeed